src/overlays/actors/ovl_Bg_Kin2_Bombwall/z_bg_kin2_bombwall.c

```diff
@@ -3,20 +3,33 @@
  * Overlay: ovl_Bg_Kin2_Bombwall
  * Description: Ocean Spider House - Bombable Wall
  */
-
 #include "z_bg_kin2_bombwall.h"
+#include "objects/object_kin2_obj/object_kin2_obj.h"
 
 #define FLAGS (ACTOR_FLAG_10 | ACTOR_FLAG_10000000)
 
-#define THIS ((BgKin2Bombwall*)thisx)
+#define THIS ((BgKin2Bombwall*)(thisx))
 
 void BgKin2Bombwall_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgKin2Bombwall_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgKin2Bombwall_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgKin2Bombwall_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-#if 0
-const ActorInit Bg_Kin2_Bombwall_InitVars = {
+s32 BgKin2Bombwall_IsHitFromNearby(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SpawnEffects(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SetupWait(BgKin2Bombwall*);
+void BgKin2Bombwall_Wait(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SetupPlayCutscene(BgKin2Bombwall*);
+void BgKin2Bombwall_PlayCutscene(BgKin2Bombwall*, GlobalContext*);
+void BgKin2Bombwall_SetupEndCutscene(BgKin2Bombwall*);
+void BgKin2Bombwall_EndCutscene(BgKin2Bombwall*, GlobalContext*);
+
+extern Gfx D_06000128[]; // debris
+extern Gfx D_060002C0[]; // wall crack
+extern Gfx D_06000360[]; // bombable wall
+extern CollisionHeader D_06000490;
```

Comment:
These externed symbols need to be replaced with the proper names from the object, as has been done in the other files you edited.

Reply:
Renamed in the next commit. I just replaced the D_06.... with the proper names where they were used and removed the extern's.